src/uu/rm/src/rm.rs

```diff
@@ -447,6 +465,10 @@ fn prompt_write_protected(path: &Path, is_dir: bool, options: &Options) -> bool
     }
 }
 
+fn prompt_descend(path: &Path) -> bool {
+    prompt(&(format!("rm: descend into directory {}? ", path.quote())))
```

Comment:
The `rm` here should be determined with `uucore::util_name` to match other warnings/errors/prompts.

Reply:
The rest of the rm.rs doesn't use that. Is there an example of its use?

Reply:
Oh I see, yeah maybe it's not the right time to add it then. It's what macros like `show!` and `show_warning!` use. Here's an example from `mv` using it for a prompt:

https://github.com/uutils/coreutils/blob/043c009a413edeb45013a73817dd5d303a162635/src/uu/mv/src/mv.rs#L277-L282

Could be a nice next PR to update `rm` to use it if you want.

Reply:
I have a branch where I am doing a rewrite of the rm prompts to hopefully fix rm3.sh test. So I can change it there

Reply:
In my other branch, I finally got rm3.sh test to pass... at least on UNIX systems because Windows is funny. That branch still depends on this PR pushing because rm3.sh does depend on descending